src/core/EdFi.DataManagementService.Core/ApiSchema/ApiSchemaValidator.cs

```diff
@@ -48,7 +48,7 @@ void ValidationErrorsFrom(EvaluationResults results)
             {
                 var propertyPathAndName = "$.";
 
-                if (detail.InstanceLocation != null && detail.InstanceLocation.Segments.Length != 0)
+                if (detail.InstanceLocation != null && detail.InstanceLocation.Count != 0)
```

Comment:
Latest version of this library get rid of the public `Segments` - instead, the `JsonPointer` object is _itself_ iterable, yield _string_ segments.